lib/controllers/share.js

```diff
@@ -67,6 +67,17 @@ function resolveGist(serviceOptions, id) {
         }
     });
 }
+/*
+  Generate short ID by hashing body, converting to base62 then truncating.
+ */
+function shortId(body, length) {
+    console.time('shortid');
+    var base62 = require("base-x")('0123456789abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ');
+    var buf = Buffer.from(body, 'utf-8');
+    var b = base62.encode(require('sha1')(buf));
+    console.timeEnd('shortid');
+    return b.slice(0, length || 16);
```

Comment:
Hmm a SHA1 is only 20 bytes.  It certainly shouldn't balloon to 54 characters when base62 encoded.  Is the `sha1` function returning a hex string instead of a buffer of 20 bytes maybe?